test.js

```diff
@@ -0,0 +1,66 @@
+// 1. Programming Basics
+// 'Spaghetti', 'Lasagne', 'Beans on toast', 'Porridge', 'Tomato soup', 'Salmon'
+// Create an array of the dishes listed above. Loop through the array using aforloop.
+// 1.Log out all the dishes.
+// 2.Log out a randomly chosen dish (randomly generated  index).
+
+const dishes = [
+  "Spaghetti",
+  "Lasagne",
+  "Beans on toast",
+  "Porridge",
+  "Tomato soup",
+  "Salmon",
+];
+function getDishes() {
+  for (i = 0; i < dishes.length; i++) {
+    console.log(dishes[i]);
+  }
+}
+getDishes();
+
```

Comment:
Good first part 👍🏽 

The second part of the task is not done:
` 2.Log out a randomly chosen dish (randomly generated  index).`

You could reconsider the naming. `getDishes` sounds like you are fetching dishes. Is that what the function does? 🤓